rs/nns/governance/src/neuron/types.rs

```diff
@@ -1359,6 +1394,7 @@ impl TryFrom<Neuron> for DecomposedNeuron {
             dissolve_state,
             visibility,
             voting_power_refreshed_timestamp_seconds,
+            recent_ballots_next_entry_index: recent_ballots_next_entry_index.map(|x| x as u32),
```

Comment:
It would be more hygienic to use try_from instead of as and handle the error, but it's unlikely to overflow in practice.

Reply:
I'm going to accept the risk here and add a comment explaining.  This value can't be very large b/c of what its function is

Reply:
oh nevermind this is already in a try_from, I will do what you suggest there.